### PR TITLE
Chat API homepage

### PR DESCRIPTION
**Pull request by AaronForinton**
- Adding a basic index page to go alongside the Chat API docs added in #5

### CODE REVIEW

File: integrations/web-chat/introduction.mdx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+- [SMS](sms)
+- [In-app chat](in-app-chat)
+- [WhatsApp](whatsapp)
+
+## Chatting with the API
+
+1. **Initiate conversation**: Send a POST `/chat/create` request with required headers.
+2. **Send and receive messages**: Use the `/chat/respond` endpoint for ongoing interaction.
+3. **Validation**: Requests are validated for API key, token, and payload.
+4. **Conversation termination**: Use the `end_conversation` flag in responses to identify when a session ends.
```
Comment (josh-hook) [inline integrations/web-chat/introduction.mdx:25]: Are we elaborating on these in a separate page?

File: integrations/web-chat/introduction.mdx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  {
+    "Action": "PLATFORM:PROJECT-eb85e04a:POST:/v1/PLATFORM/PROJECT-eb85e04a/chat/create",
+    "Effect": "ALLOW"
+  },
+  {
+    "Action": "PLATFORM:PROJECT-eb85e04a:POST:/v1/PLATFORM/PROJECT-eb85e04a/chat/respond",
+    "Effect": "ALLOW"
+  }
+]
+```
```
Comment (josh-hook) [inline integrations/web-chat/introduction.mdx:69]: Are these internal or external docs? - I don't think we need to show our auth structure externally

File: integrations/web-chat/introduction.mdx (hunk 1 above)
Comment (josh-hook) [reply integrations/web-chat/introduction.mdx:25]: Also if we're moving to a separate page then AWS do this sort of thing quite well. They define a problem then walk through the solution with diagrams, code blocks and lots of explanation.
e.g. https://aws.amazon.com/blogs/contact-center/make-your-amazon-connect-chat-experience-more-engaging-with-custom-participants-and-generative-ai-powered-chatbots/

They then have separate pages also for each API endpoint (which the solution/example docs all link back to) which detail the request and response https://docs.aws.amazon.com/connect/latest/APIReference/API_CreateParticipant.html

File: integrations/web-chat/introduction.mdx (hunk 2 above)
Comment (AaronForinton) [reply integrations/web-chat/introduction.mdx:69]: external -- still figuring this stuff out for the first time

File: integrations/web-chat/introduction.mdx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Use the Chat API to integrate PolyAI agents into existing <Tooltip tip="Non-voice chat systems, including live in-browser chats and widgets, as well as text and email conversations.">Webchat</Tooltip>
+systems, and use its endpoints to get programmatic access to your automated chat interactions. This API structures non-voice web
+integration conversations, messages, and chat sessions using a simple set of REST endpoints.
+
+The Chat API powers multiple PolyAI Webchat integrations:
+
+- [Email](email)
+- [SMS](sms)
+- [In-app chat](in-app-chat)
+- [WhatsApp](whatsapp)
```
[the remark this answers is not in the document]
Comment (AaronForinton) [reply integrations/web-chat/introduction.mdx:18]: not out of the loop, my presumption

**Pull request closed without merging**